Specifications/Concepts.Serialization.Protobuf/for_Serializer/when_serializing_concept_of_date_time.cs

```diff
@@ -28,7 +29,7 @@ class type_for_serialization
             deserialized = serializer.FromProtobuf<type_for_serialization>(bytes);
         };
 
-        It should_hold_the_correct_value = () => deserialized.concept.ShouldEqual(original.concept);
+        It should_hold_the_correct_value = () => deserialized.concept.Value.LossyEquals(original.concept.Value).ShouldBeTrue();
```

Comment:
I would have preferred to have captured this in an MSpec ShouldExtension but we don't really have a place for these and I don't want to create a whole new project, so it's ok.  We should consider whether we want to make DateTime and DateTimeOffset concepts with this equals and comparison built in.